### PR TITLE
[#12919] Allow adding emoji in status input even if it exceeds limit

### PR DESCRIPTION
**Pull request by rasom**
fixes (partially) #12919

Screenshot 2021-12-22 at 17 10 09

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/status/new/views.cljs [hunk 1]
```diff
@@ -53,7 +54,8 @@
         input-text (re-frame/subscribe [:chats/timeline-chat-input-text])
         sending-image (re-frame/subscribe [:chats/timeline-sending-image])]
     (fn []
-      (let [{:keys [uri]} (first (vals @sending-image))]
+      (let [{:keys [uri]} (first (vals @sending-image))
+            text-lenght (count @input-text)]
```
Comment (Parveshdhull) [inline src/status_im/ui/screens/status/new/views.cljs:58]: lenght - typo ?

Comment (bitgamma) [inline src/status_im/ui/screens/status/new/views.cljs]: text-length and message-max-length are misspelled

**Review by bitgamma — APPROVED** [review]

**Review by Parveshdhull — APPROVED** [review]

File: src/status_im/ui/screens/status/new/views.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im/ui/screens/status/new/views.cljs]: oh jeez... I will fix it, thanks

Comment (rasom) [reply src/status_im/ui/screens/status/new/views.cljs]: fixed

**Comment by pavloburykh** [comment]
## What has been tested: 

1. For IOS now it is possible to add emoji even if exceeds a limit and it does not result in deletion of value in input. At the same time user is unable to post message until he enters the allowed amount of 600 symbols.

2. For Android nothing changed and behaviour is correct: it is still impossible to exceed input limit and enter emoji if there is not enough symbols left available for entering. This bug  https://github.com/status-im/status-react/issues/12919  is reproducible only for IOS

**Pull request merged**